Automattic/wp-calypso
# Remove business plugins (cleanup of #3079)

In the context of removing the eCommerce feature from the Business plan upgrade, the plugins page becomes available to Jetpack sites only. The "Plugins" menu entry is being removed on https://github.com/Automattic/wp-calypso/pull/3079

This PR removes references to plugins related to the Business plan (eCommerce plugins) and display the "Plugins not supported" page at `/plugins/:site` for all wp.com sites.

#### Testing instructions
- `git fetch -a && git checkout remove/business-plugins && make run`

##### Test 1 (Wp.com)
- Go to Calypso and select a wordpress.com site. Check that the Plugins menu is not displayed (because the site does not have any plugin not because #3079 changes are included in this PR)
- Go to `/plugins/:site` and check that the "Plugins not supported" page is displayed
- Go to `/plugins/browse/:site` and check that the "Plugins not supported" page is displayed
- Go to `/plugins/akismet/:site` and check that the "Plugins not supported" page is displayed
- Go to `/plugins` and check that only your jetpack sites are counted.

##### Test 2 (Jetpack, test that nothing broke)
- Go to Calypso and select a jetpack site. Check that the Plugins menu is displayed
- Go to the Plugins page and check that your plugins are all there
- Try to install, activate, deactivate, remove a plugin, (and update if you have one which is not updated) it should work without any error
- Back to the main Plugins page, click on the Edit All and try to bulk remove, bulk update, bulk activate bulk deactivate, bulk autoupdate and bulk disable autoupdate your plugins, it should work without any error

Code Review Instructions
- Review first commit first and check that all mentions and related features of wpcom plugins were removed
- Review second commit

Reviewing

cc: @fabianapsimoes @drewblaisdell @scruffian
